README.md

```diff
@@ -8,7 +8,7 @@ This project provides a home to Akka Streams connectors to various technologies,
 Documentation
 -------------
 
-Latest [documentation](http://developer.lightbend.com/docs/alpakka/latest/).
+See [Releases](https://github.com/akka/alpakka/releases) page for the reference and API documentation.
```

Comment:
I like this, because it allows us to have one less step when cutting a release. And releases page has some useful information.